docs/display_panels.rst

```diff
@@ -2,147 +2,103 @@
 
-You can reset an axis to "auto" by double clicking on it.
-
-The display limits and display channels may be specified or unspecified, in which case they are "auto" calculated, meaning that they will automatically adjust to the lowest and highest values of the data (once it has been converted to scalar) in the plot.
-
-The y-axis can be displayed as linear or logarithmic by clicking on the :guilabel:`Log Scale (Y)` checkbox in the :guilabel:`Line Plot Display` section of the Inspector.
-
-The line plot can display calibrated or uncalibrated values. You can change what is displayed by changing the :guilabel:`Display` setting in the :guilabel:`Calibrations` section of the Inspector.
-
-You can also change calibration in the Inspector. However, if the data is a result of a computation or acquisition, the calibration will get recalculated/reset when the data is recomputed/reacquired.
-
-The Info panel (see :ref:`Info Panel`) shows the value of the data underneath the cursor.
-
-You can add interval graphics to the line plot by dragging across the plot or by using the menu item :menuselection:`Processing --> Add Interval Graphic`.
-
-You can add a channel graphic to the line plot by using the menu item :menuselection:`Processing --> Add Channel Graphic`.
-
-You can use the mouse to move and resize interval graphics. You can also use the mouse to move channel graphics. You can use the Inspector to adjust the endpoints of the interval graphics or position of the channel graphics.
+Line Plot Layers
+----------------
+Line plots can be layered to show multiple data items on the same plot. To layer line plots, drag a line plot data item from the data panel onto the graph portion of a displayed line plot. Do not drop the data item onto the sides of the display panel or the data item will replace the line plot in the display panel. Line plots can only be layered if the calibrated units match those of the first data item.
 
-You can use the :kbd:`Left` and  :kbd:`Right` arrow keys to nudge an interval graphic or channel graphics. Holding the :kbd:`Shift` key while nudging the graphic will make it move farther.
+.. image:: graphics/line_plot_layers_infographic.png
+    :width: 792
+    :alt: Adjust Line Plot Layers
 
-You can press the :kbd:`Delete` key with a interval graphic or channel graphic selected to delete the it.
+Each layer of a line plot will have a unique fill color by default. To change the fill color, stroke color, and stroke width of each layer, use the Line Plot Display Layers subsection of the Inspector panel. Clicking on the color box next to Fill Color or Stroke Color will open the color selection window. To input color at text, you can use RGB values such as rgb(100, 50, 200), hex values such as #55AAFF, or a web-defined color such as Blue. Change the transparency of a color by either inputting text with a transparency code like rgb(100, 50, 200, .5) or #55AAFF80, or by using the opacity sliders in the pop-up color selection panel.
```

Comment:
As opposed to bullet points, this section uses paragraphs, which, in my opinion, works better. However, even with the paragraphs, the user may have trouble navigating quickly.

What about using sub-headers instead of bullet points for everything on this page? Even if the sub-header is just one short sentence, it might work nicely. The sub-header should use a header level that make it unobtrusive, but easy to navigate. Maybe just bold/italic? You might have to experiment with this if you try it.

Reply:
Definitely an improvement! Some of the headers need some tweaks, though.